Automattic/wp-calypso
# WooCommerce: Add Powered by WooCommerce footer to the WooCommerce installation page

#### Changes proposed in this Pull Request

* Adds the "Powered by WooCommerce" footer to the `/woocommerce-installation` page (as was previously done in `/store`)

#### Testing instructions



With an Atomic site with a Business plan and without WooCommerce installed

1. Click on the "WooCommerce" menu item
2. Verify that the "Powered by WooCommerce" footer is shown
3. Verify that clicking on the footer navigates to https://woocommerce.com

<img width="1275" alt="Screen Shot 2021-03-03 at 12 58 35" src="https://user-images.githubusercontent.com/2098816/109850834-f4a23180-7c20-11eb-87f9-48241e4c086b.png">
